IdentityCore/src/webview/systemWebview/ios/MSIDSafariViewController.m

```diff
@@ -66,8 +66,9 @@ - (instancetype)initWithURL:(NSURL *)url
     {
         _startURL = url;
         _context = context;
-        
-        _safariViewController = [[SFSafariViewController alloc] initWithURL:url entersReaderIfAvailable:NO];
+        SFSafariViewControllerConfiguration *config = [[SFSafariViewControllerConfiguration alloc] init];
+        config.entersReaderIfAvailable = NO;
```

Comment:
Just curios & for my learning - what issues were you hoping to address with this change?

Reply:
@NerevarineRule This one is a trivial & interesting change. As part of making ADAL compatible with iOS 14, xcode 12 we bumped the target deployment to iOS 11. The above method initWithURL entersReaderIfAvailable: is deprecated in iOS 11 so replaced it with its replacement. ADAL does not support SFSafariViewController so this change wasn't really necessary as this code is not included in ADAL during build. But during ADAL's release we need to run lint command on ADAL.podspec to validate the pod. During this process it fails validation saying that the above method is deprecated.